lib/custom/src/MShop/Service/Provider/Payment/Stripe.php

```diff
@@ -21,6 +22,99 @@ class Stripe
 	extends \Aimeos\MShop\Service\Provider\Payment\OmniPay
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
+
+	private $beConfig = array(
+		'address' => array(
+			'code' => 'address',
+			'internalcode'=> 'address',
+			'label'=> 'Send address to payment gateway too',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'authorize' => array(
+			'code' => 'authorize',
+			'internalcode'=> 'authorize',
+			'label'=> 'Authorize payments and capture later',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'testmode' => array(
+			'code' => 'testmode',
+			'internalcode'=> 'testmode',
+			'label'=> 'Test mode without payments',
+			'type'=> 'boolean',
+			'internaltype'=> 'boolean',
+			'default'=> '0',
+			'required'=> false,
+		),
+		'apiKey' => array(
+			'code' => 'apiKey',
+			'internalcode'=> 'apiKey',
+			'label'=> 'API key',
+			'type'=> 'string',
+			'internaltype'=> 'string',
+			'default'=> '',
+			'required'=> true,
+		),
+		'publishableKey' => array(
+			'code' => 'publishableKey',
+			'internalcode'=> 'publishableKey',
+			'label'=> 'publishable key',
+			'type'=> 'string',
+			'internaltype'=> 'string',
+			'default'=> '',
+			'required'=> true,
+		),
+	);
+
+	protected $feConfig = array(
+
+		'paymenttoken' => array(
```

Comment:
What about the payment token? Is that the token we need to store for executing the payment? When exactly is the payment executed?

Reply:
When we send card details to Stripe, we take token as Stripe's answer. Then I put the token at this hidden field 'paymenttoken', and this is only one value we have to send to Stripe later via Omnipay. So this field is necessary.

Reply:
Payment will be executed when we'll send token to Stripe by Omnipay.